{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,41 +1,46 @@
-FROM python:3.6-alpine3.7
-
-ENV PYTHONUNBUFFERED=1
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM dddecaf/wsgi-base:master
 
 ENV APP_USER=giraffe
 
 ARG UID=1000
 ARG GID=1000
 
-ARG CWD=/app
-
-ENV PYTHONPATH=${CWD}/src
+ARG CWD="/opt/app"
```

Comment:
Is there a good reason to change this?

Reply:
No, it's mostly indecision as I mentioned earlier. I had used `/opt` in the base image for the requirements file so I thought it makes some sense to build on that but I'm also happy to revert to `/app`.